src/components/home-page/user-gallery-overview/OnsaleItem.vue

```diff
@@ -0,0 +1,741 @@
+      this.wasLiked = true;
+      this.likes++;
+    }
+  }
+
+  async endAuction() {
+    if (!this.auction) {
+      return;
+    }
+
+    this.auctionSystem = new AlgoPainterAuctionSystemProxy(this.networkInfo);
+
+    this.displayEndAuctionStatus = true;
+    this.endAuctionStatus = EndAuctionStatus.EndAuctionAwaitingInput;
+
+    await this.auctionSystem.endAuction(
+      this.auction.index,
+      this.account,
+    ).on('error', () => {
+      this.endAuctionStatus = EndAuctionStatus.EndAuctionError;
+    }).on('transactionHash', () => {
+      this.endAuctionStatus =
+        EndAuctionStatus.EndAuctionAwaitingConfirmation;
+    });
+
+    this.endAuctionStatus = EndAuctionStatus.AuctionEnded;
+  }
+
+  async cancelAuction() {
+    this.auctionSystem = new AlgoPainterAuctionSystemProxy(this.networkInfo);
+
+    this.displayCancelAuctionStatus = true;
+    this.deleteAuctionStatus = DeletingAuctionStatus.DeleteAuctionAwaitingInput;
+
+    await this.auctionSystem.cancelAuction(this.auction.index, this.account).on('transactionHash', () => {
+      this.deleteAuctionStatus = DeletingAuctionStatus.DeleteAuctionAwaitingConfirmation;
+    }).on('error', () => {
+      this.deleteAuctionStatus = DeletingAuctionStatus.DeleteAuctionError;
+    });
+
+    this.deleteAuctionStatus = DeletingAuctionStatus.AuctionDeleted;
+  }
+
+  onCloseEndAuctionDialog() {
+    this.displayEndAuctionStatus = false;
+  }
+
+  onCloseCancelAuctionDialog() {
+    this.displayCancelAuctionStatus = false;
+
+    if (this.deleteAuctionStatus === DeletingAuctionStatus.AuctionDeleted) {
+      this.$q.notify({
+        type: 'positive',
+        message: this.$t('dashboard.auctionPage.cancelAuctionStatuses.deleteAuctionDeleted'),
+      });
+    }
+  }
+
+  openAuctionModal() {
+    void this.$store.dispatch('auctions/openAuctionModal');
+  }
+}
+
+</script>
+
+<style lang="scss" scoped>
+
+.cursor-pointer {
+  cursor: pointer;
+}
+
+.teste {
+  width: 100px;
+  height: 100px;
+  background-color: red;
+}
+
+.bidBack{
+  text-align: unset;
+  height: 50px;
+  width: 50px;
+  border-radius: 50%;
+  background-color: $primary;
+  font-size: 9px;
+  margin-left: 9px ;
+}
+
+.pirs{
+  text-align: unset;
+  height: 50px;
+  width: 50px;
+  border-radius: 50%;
+  background-color: $primary;
+  font-size: 9px;
+  margin-left: 9px ;
+  margin-right: -10px
+}
+
+.pirs-bidback{
+  margin-top: -13px;
+  margin-right: 0px;
+  background: none;
+  width: 100%;
+}
+
+  .ellipsis{
+    text-overflow: ellipsis;
+    overflow: hidden;
+    white-space: nowrap;
+    text-align: left;
+    max-width: 120px;
+  }
+
+.users {
+  .q-avatar:not(:first-child) {
+    margin-left: -8px;
+  }
+}
+
+.art-image {
+  border-radius: 5px;
+  width: 300px;
+  height: 300px;
+}
+
+.price {
+    color: $positive;
+    font-size: 1.1rem;
+    font-weight: bold;
+  }
+
+  .highest-bid {
+    font-size: 1.1rem;
+    display: flex;
+    align-items: center;
+    b {
+      margin-left: 5px;
+    }
+  }
```

Comment:
```suggestion
.price {
  color: $positive;
  font-size: 1.1rem;
  font-weight: bold;
}

.highest-bid {
  font-size: 1.1rem;
  display: flex;
  align-items: center;

  b {
    margin-left: 5px;
  }
}
```
Lembre de verificar indentação.

Reply:
foi corrigido.